src/main/java/com/google/cloud/tools/appengine/operations/DevServer.java

```diff
@@ -148,6 +151,39 @@ public void run(RunConfiguration config) throws AppEngineException {
     }
   }
 
+  /**
+   * Simple helper function to try and extract the version specified. Very limited validation done
+   * to ensure that the projectJdkVersion is set properly and the value is decoded with best effort.
+   * Expected values should follow the `java.specification.version` syntax.
+   *
+   * <p>Format should be 1.8 or 9+ as the project's min version is Java 8
```

Comment:
Would you add explanation that the return value is 8 for "1.8" and 11 for "11.0.0".

Reply:
Added more info for this doc and that the value *should* come in as either 1.8 or 9+. Added a test for `11.0.0`